Unix/http/httpclient.c

```diff
@@ -813,6 +817,37 @@ static Http_CallbackResult _ReadData(
         _WriteTraceFile(ID_HTTPCLIENTRECVTRACEFILE, (char*)(handler->recvPage + 1), handler->receivedSize);
     }
 
+
+    if (!handler->ssl && handler->encrypting)
+    {
+
+#if ENCRYPT_DECRYPT
```

Comment:
Why a compile-time conditional for this?

If the compile-time conditional is disabled, but we're supposed to encrypt, then this if will fall through without returning any kind of status/result. Is this correct?